Makefile.am

```diff
@@ -274,6 +277,14 @@ EXTRA_DIST += \
   NON-AUTOTOOLS-BUILD \
   HACKING
 
+# These are support files for building with Bazel or Zig
+
+EXTRA_DIST += \
+  BUILD.bazel \
+  MODULE.bazel \
+  WORKSPACE.bazel \
+  build.zig
```

Comment:
Within reason, I feel that users grabbing the tarball should be getting the same experience as users who check out from git (except that the tarball users don't need to run autogen.sh). So I'm added some missing files to the tarball.

Reply:
Carefully here, since this specific file are usually updated in non backward compatible ways, and are (I presume) used by some external build process that most likely only uses the git files.